src/telliot_feeds/reporters/tips/listener/funded_feeds_filter.py

```diff
@@ -85,17 +85,17 @@ async def price_change(self, query_id: bytes, value_before: float) -> Optional[f
         else:
             logger.info(f"No Api source found for {query_tag} to check priceThreshold")
             return None
+        if query_id not in self.prices:
```

Comment:
I'm wondering if this solution will create a different problem:

Once it encounters a query id, it will stop doing api calls associated w/ it. So after an hour, won't it be using a potentially stale price for the tips calculation?

Reply:
the function this line is in only gets called in one function that sets self.prices to an empty dict and checks price in a for loop that doesn't carry the price after the loop ends so its reset every telliot reporting loop.